titus-api/src/main/java/com/netflix/titus/api/jobmanager/TaskAttributes.java

```diff
@@ -22,15 +22,19 @@
      */
     public static final String TASK_ATTRIBUTES_AGENT_REGION = "agent.region";
     public static final String TASK_ATTRIBUTES_AGENT_ZONE = "agent.zone";
+    public static final String TASK_ATTRIBUTES_AGENT_AMI = "agent.ami";
     public static final String TASK_ATTRIBUTES_AGENT_ASG = "agent.asg";
     public static final String TASK_ATTRIBUTES_AGENT_HOST = "agent.host";
+    public static final String TASK_ATTRIBUTES_AGENT_CLUSTER = "agent.cluster";
     public static final String TASK_ATTRIBUTES_AGENT_INSTANCE_ID = "agent.instanceId";
     public static final String TASK_ATTRIBUTES_AGENT_ITYPE = "agent.itype";
+    public static final String TASK_ATTRIBUTES_AGENT_RES = "agent.res";
```

Comment:
minor: I know it is not related to your PR, but what is `agent.res`? A comment here may help

Reply:
Adding a comment now in another PR. This attributes holds agent ENI resources.